### PR TITLE
Implement sentry error reporting

### PR DESCRIPTION
**Pull request by ftsell**
With this the someone can configure the software to report errors to a Sentry instance e.g. [sentry.mafiasi.de](https://sentry.mafiasi.de)

### CODE REVIEW

**Comment by Flova** [comment]
The backend looks good

File: lecture2gether-vue/src/main.ts [hunk 1]
```diff
@@ -3,13 +3,34 @@ import App from './App.vue';
...
 
 Vue.config.productionTip = false;
 
-new Vue({
-    router,
-    store,
-    vuetify,
-    render: (h) => h(App),
-}).$mount('#app');
+// Fetch settings because we can only initialize sentry before Vue      TODO Improve because loading delay
```
Comment (Flova) [inline lecture2gether-vue/src/main.ts:12]: Whats up with this todo? Is it necessary?

Comment (ftsell) [reply lecture2gether-vue/src/main.ts:12]: The todo signals that this should be improved but I don't know how so that's why it's still there.

**Comment by JanGut** [comment]
TODO:

- [ ] Add env vars to readme

**Pull request merged**